src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/Watch.cs

```diff
@@ -39,15 +39,15 @@ public void CustomizeView(Watch nodeModel, NodeView nodeView)
             this.watch = nodeModel;
             this.syncContext = new DispatcherSynchronizationContext(nodeView.Dispatcher);
 
-            var watchTree = new WatchTree();
```

Comment:
Is any of this code new to 2.15 that causes this to fail only in 2.15?

Reply:
Hard to tell by looking at the code...multiple things can contribute to this bug
The critical area (IMO) is the `Loaded` event ...that assumes a DataContext exists. That part of the code has been there for a while.

The loaded event can be triggered multiple times with/without data context. Looks like at least in 2.15...a Loaded event is triggered without data context (when opening the .dyn attached to the jira task)
This seems to be acceptable (from what I have read) but we should not always assume the context exists.